app/i18n/translations/original.json

```diff
@@ -1094,10 +1094,10 @@
   "tickets.startAutoBuyerConfirmation.description": "Decrediton must remain running for tickets to be automatically purchased.",
   "tickets.startAutoBuyerHeader": "Ticket buyer is now running.",
   "tickets.stopAutoBuyerHeader": "Ticket buyer is now stopped.",
-  "tickets.tab.mytickets": "History Tickets",
+  "tickets.tab.mytickets": "Tickets History",
```

Comment:
Ticket

Reply:
Fixed. Thanks! Not sure how I missed this one.